src/server/img_storage.rs

```diff
@@ -121,7 +121,7 @@ impl Default for StoreImage {
 #[derive(Debug)]
 pub struct StoredImage {
     pub url: uri::Uri,
-    pub object: Object,
+    pub object: Option<Object>,
```

Comment:
we can probably drop this. It's not really used.

Reply:
woops, almost forgot this

Reply:
And then I actually did forget this (forgot to push it)